### DISCUSSION TITLE
Spatial Info Issue When Loading MODFLOW 6 for PEST++ in Flopy

### DISCUSSION THREAD

**Discussion by Omowumieru** [modflowpy/flopy#2311] [Q&A] [answered]
Hi,

I successfully built a MODFLOW 6 model in Flopy with the spatial extent of the model grid set correctly using the model's shapefile and:

**sgr = flopy.discretization.StructuredGrid(delc=delc*np.ones(nrow), delr=delr*np.ones(ncol), top=ztop, botm=zbot, xoff=shpgeom.bounds[0], yoff=shpgeom.bounds[1])**

The model runs fine, and the output plots have the correct coordinate extent and CRS. However, when I load the model nam file with flopy.mf6.MFSimulation.load() to run PEST++ (pyemu), the spatial extent is wrong and needs to be redefined.

Is there a way in Flopy to ensure the spatial info is imported correctly when loading the model, or am I doing something wrong?

Thank you!

**Comment by martclanor** [comment, marked as answer]
I think during the creation of your discretization object (`ModflowGwfdis`, `ModflowGwfdisv` or `ModflowGwfdisu`), you can pass `xorigin` and `yorigin` as kwargs with their corresponding values.

**Reply by Omowumieru** [reply]
Thank you!!! It worked.
